docs/SUMMARY.md

```diff
@@ -179,10 +200,10 @@
   * [License](project-overview/license.md)
 * [Careers & Open Positions](career-and-open-positions/README.md)
   * [Senior Software Engineer](career-and-open-positions/senior-software-engineer.md)
-  * [Founding Developer Evangelist](career-and-open-positions/founding-developer-evangelist.md)
-  * [Senior Technical Writer / Editor in Chief](career-and-open-positions/technical-content-writer.md)
-  * [Senior Technical Recruiter](career-and-open-positions/recruiter.md)
-  * [Head of Lead Generation](career-and-open-positions/head-of-lead-generation.md)
+  * [Senior Product Manager](career-and-open-positions/senior-product-manager.md)
```

Comment:
is this up to date with master? just asking since these changes are unrelated 🤔

Reply:
I grabbed it from master because of merge conflicts that seemed to be irrelevant.  Looked like just a tiny local edit happened in our branch, causing conflict.  I kept what was in master.